addons/project/static/src/js/right_panel/project_right_panel.js

```diff
@@ -39,14 +39,18 @@ export default class ProjectRightPanel extends LegacyComponent {
             ...options,
             'noSymbol': true,
         });
+
+        const array = valueFormatted.split('.');
```

Comment:
I tried to use the digit option, but sadly it does not matter because the formatFloat use by default the currency format, and then the manual given precision. So added a small function 
removeDecimals(value) {
        return formatFloat(value, { digits: [false, 0] });
    }
  which i then call after the call to formatMonetary in order to remove the decimal. it looks like this :
  const valueFormatted = formatMonetary(value, {
            currencyId: this.state.data.currency_id,
            ...options,
            'noSymbol': true,
        });
        const valueFormattedToInteger = this.removeDecimals(value);
  Is that a better way to do, or schould i change the default selection insinde the formatFloat function of formatters.js ?

Reply:
Good catch, `formatMonetary` gives precedence to the currency digits indeed, and it seems intentional:
https://github.com/odoo/odoo/blob/48ef812a635f70571b395f82ffdb2969ce99da9e/addons/web/static/tests/views/fields/formatters_tests.js#L255-L259

I'm not a big fan of using both `formatMonetary` and `formatFloat`, because that's basically using `formatFloat` with different options two times in a row in our case.

I also see that contrary to what I was thinking, it doesn't localize the decimal separator, so I guess we could just use `formatFloat` then. :thinking: